File: exchange/Exchange.Web.BusinessLogic/Models/UserModel.cs
using Exchange.Web.BusinessLogic.Models.Base;
using System.Collections.Generic;

namespace Exchange.Web.BusinessLogic.Models
{
    public class UserModel : BaseModel
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Phone { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string OneSignalId { get; set; }
        public ICollection<PhotoModel> Photos { get; set; } = new List<PhotoModel>();
    }
}

File: exchange/exchange/exchangeClientApp/src/app/Models/user.model.ts
import { PhotoModel } from './photo.model';
import { BaseModel } from './base.model';

export class UserModel extends BaseModel {
  firstName: string;
  lastName: string;
  phone: string;
  country: string;
  city: string;
  email: string;
  password: string;
  oneSignalId: string;
  photos: PhotoModel[];

  constructor(model: UserModel) {
    super(model);
    this.firstName = model.firstName;
    this.lastName = model.lastName;
    this.phone = model.phone;
    this.country = model.country;
    this.city = model.city;
    this.email = model.email;
    this.password = model.password;
    this.oneSignalId = model.oneSignalId;
    this.photos = model.photos;
  }
}
